core/src/test/java/org/apache/iceberg/TestTableMetadata.java

```diff
@@ -108,7 +108,8 @@ public void testJsonConversion() throws Exception {
         7, ImmutableList.of(TEST_SCHEMA, schema),
         5, ImmutableList.of(SPEC_5), SPEC_5.lastAssignedFieldId(),
         3, ImmutableList.of(SORT_ORDER_3), ImmutableMap.of("property", "value"), currentSnapshotId,
-        Arrays.asList(previousSnapshot, currentSnapshot), snapshotLog, ImmutableList.of(), ImmutableList.of());
+        Arrays.asList(previousSnapshot, currentSnapshot), snapshotLog, ImmutableList.of(),
+        ImmutableMap.of(), ImmutableList.of());
```

Comment:
is this line wrapping correct?

Reply:
Ah no, it should be fixed, only the last arg should be on a new line